cfpb/consumerfinance.gov
# Show products on enforcement action pages

1. Show the "Products" field on the sidebar of enforcement action pages, when product data exist
2. Add "Fair Lending" to the list of possible products in the enforcement page metadata
3. Add a script that makes one-time corrections and updates to the product data
4. Remove old one-time scripts that are now outdated


## How to test this PR

1. Run this branch
2. Visit an enforcement action page, such as http://localhost:8000/enforcement/actions/toyota-motor-credit-corporation/ and see that the "Products" heading shows up in the sidebar with product content
2. Contact me to get the CSV for the data update
3. Run `cfgov/manage.py runscript enforcement-add-fair-lending --script-args path/to/csv`
4. Reload the enforcement action page, see that the product list has been updated


## Screenshots

![Screen Shot 2021-03-09 at 5 53 43 PM](https://user-images.githubusercontent.com/4295388/110549687-74883a00-8100-11eb-949c-0d87fd2a4ec4.png)


## Notes and todos

- After this PR is merged, we will need to run the script, then delete it


## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
